lib/adyen/services/payments.rb

```diff
@@ -17,6 +17,10 @@ def initialize(client, version = DEFAULT_VERSION)
         :cancel_or_refund,
         :adjust_authorisation,
         :donate,
+        :get_authentication_result,
+        :technical_cancel,
+        :void_pending_refund,
+        :retrieve3ds2_result
```

Comment:
Let's add an underscore before 3ds2 to keep it consistent - so `retrieve_3ds2_result`.